run-server.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/sh -e
 
-if [ "z$HOST" = "z" ]; then
+if [ -z "$HOST" ]; then
```

Comment:
You don't get anything from wrapping it in brackets. It's only useful when concatenating with another string so that bash doesn't expand your variable.